prowler/__main__.py

```diff
@@ -280,9 +282,29 @@ def prowler():
             )
             sys.exit(1)
 
+    # Outputs
+    # TODO: this part is needed since the checks generates a Check_Report_XXX and the output uses Finding
+    # This will be refactored for the outputs generate directly the Finding
+    finding_outputs = []
+    for finding in findings:
+        finding_outputs.append(Finding.generate_output(global_provider, finding))
+
     if args.output_formats:
         for mode in args.output_formats:
+
+            if "csv" in mode:
+                # Generate CSV Finding Object
+                filename = f"{global_provider.output_options.output_directory}/{global_provider.output_options.output_filename}{csv_file_suffix}"
```

Comment:
Line exceeded the length limit.

```suggestion
                filename = (f"{global_provider.output_options.output_directory}/"
                            f"{global_provider.output_options.output_filename}{csv_file_suffix}")
```

Reply:
Done